src/matcher/commonjs.js

```diff
@@ -2,13 +2,13 @@
 
 var regexToArray = require('../util/regex-to-array');
 
-function Matcher() {}
-Matcher.prototype = {
-  match: function (code) {
+module.exports = function ($fs) {
+  return function (file, code) {
     return regexToArray(/require\([\'"]([^\'"]+)[\'"]\)/g, code).map(function (imp) {
```

Comment:
Yeah, for sure. The plan is to start using an AST once we have a fairly feature complete starting point for what we need.